src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -1573,78 +1573,74 @@ public void canPageAllItems()
     assertThat(secondPage.getInteger("totalRecords"), is(5));
   }
 
-  @Test
-  public void canSearchForItemsByBarcodeWithLeadingZero()
-    throws MalformedURLException,
-    InterruptedException,
-    ExecutionException,
-    TimeoutException {
+  /**
+   * Assert that the cql query returns items with the expected barcodes.
+   */
+  private void assertCqlFindsBarcodes(String cql, String ... expectedBarcodes) throws Exception {
+    Items items = findItems(cql);
+    String [] barcodes = items.getItems().stream().map(Item::getBarcode).toArray(String []::new);
+    assertThat(cql, barcodes, is(expectedBarcodes));
+    assertThat(cql, items.getTotalRecords(), is(barcodes.length));
+  }
```

Comment:
Given that these helper methods are intended to aid the readability of the tests, please move them to the bottom of the test file so they do not distract from the tests themselves.

Reply:
Moved.